math/module_argument_cplexnum.cpp

```diff
@@ -0,0 +1,52 @@
+/// c++ program to calculate the modulus and argument of a complex number
```

Comment:
please use [doxygen](https://www.doxygen.nl/manual/docblocks.html) commands for documentation. you can also refer to the file in PR #807.